src/energyplus/ForwardTranslator.cpp

```diff
@@ -979,8 +979,12 @@ namespace energyplus {
         break;
       }
       case openstudio::IddObjectType::OS_Coil_Cooling_DX: {
-        model::CoilCoolingDX dx = modelObject.cast<CoilCoolingDX>();
-        retVal = translateCoilCoolingDX(dx);
+        model::CoilCoolingDX coil = modelObject.cast<CoilCoolingDX>();
+        if (this->isHVACComponentWithinUnitary(coil)) {
+          retVal = translateCoilCoolingDXWithoutUnitary(coil);
+        } else {
+          retVal = translateCoilCoolingDX(coil);
+        }
```

Comment:
Breaking original FT into two separate methods. The "WithoutUnitary" version is the original. The new one allows the Coil:Cooling:DX object to be placed on a CoilSystem:Cooling:DX object (which is newly valid in E+); this follows the same logic as Coil:Cooling:DX:SingleSpeed, Coil:Cooling:DX:VariableSpeed, etc.